src/k8sconfiguration/README.rst

```diff
@@ -0,0 +1,74 @@
+Microsoft Azure CLI 'k8sconfiguration' Extension
+==========================================
+
+This package is for the 'k8sconfiguration' extension.
+i.e. 'az k8sconfiguration'
+
+### How to use ###
+Install this extension using the below CLI command
+```
+az extension add --name k8sconfiguration
+```
+
+### Included Features
+#### Kubernetes Configuration:
+Kubernetes SourceControl Configuration: [more info](https://docs.microsoft.com/en-us/azure/kubernetessconfiguration/)\
+*Examples:*
+
+##### Create a KubernetesConfiguration
+```
+az k8sconfiguration create \
```

Comment:
I think the command name "k8sconfiguration" is a little bit long.

Reply:
Can you explain why?  Will this cause an issue in any of the tools or process?  This is the shorter form of 'kubernetesconfiguration'.

Reply:
It's just my personal preference as a user. I don't like typing long name. 😂

Reply:
Ah, got it :)  As I mentioned earlier, it is already shortened.  So, we'd want to keep it like this.